frontend/static/quotes/code_csharp.json

```
@@ -0,0 +1,71 @@
+{
+  "language": "code_csharp",
+  "groups": [
+    [0, 100],
+    [101, 300],
+    [301, 600],
+    [601, 9999]
+  ],
+  "quotes": [
+    {
+      "text": "string aFriend = \"Bill\";\nConsole.WriteLine($\"Hello {aFriend}\");",
+      "source": "Microsoft Learn - Hello World",
+      "id": 1,
+      "length": 63
+    },
+    {
+      "text": "static int factorial(int n)\n{\n\tint result = 1;\n\tfor (int i = 1; i <= n; i++)\n\t{\n\t\tresult = result * i;\n\t}\n\treturn result;\n}",
+      "id": 2,
+      "source": "Geeks for Geeks - C# Methods",
+      "length": 123
+    },
+    {
+      "text": "public static int Fibonacci(int n) {\n\tif (n <= 2) {\n\t\treturn 1;\n\t} else {\n\t\treturn Fibonacci(n - 1) + Fibonacci(n - 2);\n}",
+      "id": 3,
+      "source": "Community Contribution - Find the nth Fibonacci Number Recursively",
```

Should I remove the Community Contribution prefix to make it shorter?

```suggestion
      "source": "Find the nth Fibonacci Number Recursively",
```